cypress/integration/chi-js/modal.spec.js

```diff
@@ -0,0 +1,182 @@
+
+    it(`modal should be closed when we click cancel button`, () => {
+      cy.get('button#modal-trigger-1').click();
+      cy.get('#modal-1')
+        .contains('Cancel')
+        .click()
+        .wait(550)
+        .should('be.not.visible');
+    });
+  });
+
+  describe('check Center modal', () => {
+    it(`modal should be open in the center of the page when we click launch demo modal button`, () => {
+      cy.get('#modal-trigger-2').click();
+      cy.get('#modal-2')
+        .parent()
+        .parent()
+        .should('have.class', '-center')
+        .find('.-close')
+        .wait(550)
+        .click();
+    });
+  });
+
+  describe('check scrollable modal', () => {
+    it(`modal should be scrollable`, () => {
+      cy.get('button#modal-trigger-3')
+        .click()
+        .wait(550);
+      cy.get('#modal-3 .chi-modal__content')
+        .should('have.class', '-mh--400')
+        .scrollTo('bottom');
+      cy.get('#modal-3')
+        .find('.-close')
+        .click()
+        .wait(550)
+        .should('be.not.visible');
+    });
+  });
+
+  describe('check scrollable modal for long content', () => {
+    it(`modal should be scrollable when long content is present`, () => {
+      cy.get('button#modal-trigger-4')
+        .click()
+        .wait(550);
+      cy.get('#modal-trigger-4 + .chi-backdrop')
+        .should('have.class', '-mh--480')
+        .scrollTo('bottom');
+      cy.get('#modal-4')
+        .find('.-close')
+        .click()
+        .wait(550)
+        .should('be.not.visible');
+    });
+  });
+
+  describe('check modal with subtitle', () => {
+    it(`modal should contain subtitle`, () => {
+      cy.get('button#modal-trigger-5').click();
+      cy.get('#modal-5')
+        .find('.chi-modal__header')
+        .children()
+        .should('have.class', 'chi-modal__subtitle');
+      cy.get('#modal-5')
+        .find('.-close')
+        .click()
+        .wait(550);
+    });
+  });
+
+  describe('check modal with inverse', () => {
+    it(`modal should contain inverse background`, () => {
+      cy.get('button#modal-trigger-6').click();
+      cy.get('#modal-6')
+        .parent()
+        .parent()
+        .should('have.class', '-inverse')
+        .find('.-close')
+        .wait(550)
+        .click();
+    });
+  });
+
+  describe('check modal has no borders', () => {
+    it(`modal should not contain border for header`, () => {
+      cy.get('button#modal-trigger-7').click();
+      cy.get('#modal-7')
+        .find('.chi-modal__header')
+        .should('have.class', '-no-border')
+        .find('.-close')
+        .wait(550)
+        .click();
+    });
+    it(`modal should not contain border for footer`, () => {
+      cy.get('button#modal-trigger-7').click();
+      cy.get('#modal-7')
+        .find('.chi-modal__footer')
+        .should('have.class', '-no-border')
+        .contains('Cancel')
+        .wait(550)
+        .click();
+    });
+  });
+  describe('check multi-step modal', () => {
```

Comment:
The context of the example in documentation is to provide markup of the next and back buttons. No functionality needs to be checked here. Let's remove the test case.

Reply:
Fixed in the latest commit